DigitalLearningSolutions.Data/DataServices/CentresDataService.cs

```diff
@@ -99,6 +99,7 @@ public IEnumerable<(int, string)> GetActiveCentresAlphabetical()
                 @"SELECT CentreID, CentreName
                         FROM Centres
                         WHERE Active = 1
+                        AND kbSelfRegister = 1

```

Comment:
Happy to put this in the db query level given that this is only used in registration, but I think we should modify the name of the method so it's clear that we're getting the centre options for delegate self registration (rather than a true active centres list, which should technically only have the Active = 1 constraint)

Reply:
Good idea! I changed the method name to account for this